### PR TITLE
env option was missing on tc task

### LINKED ISSUE #80: `tc` task  is broken by #73
related docopt is missing on `tc` task

### PR DESCRIPTION
**Pull request by msimonin**
fix #80

### CODE REVIEW

File: enos/enos.py [hunk 1]
```diff
@@ -470,14 +470,17 @@ def ssh_tunnel(env=None, **kwargs):
...
+usage: enos tc [--test] [-e ENV|--env=ENV] [-vv|-s|--silent]
 
 Enforce network constraints
 
 Options:
   --test   Test the rules by generating various reports
+  -e ENV --env=ENV     Path to the environment directory. You should
+                       use this option when you want to link a specific
+                       experiment [default: %s].
   -h --help                 Show this help message.
```
Comment (denaitre) [inline enos/enos.py]: I think we should:
* sort the option list alphabetically (i.e. `--env`, `--help`, `--test`) similarly to other help messages; 
* align the description of `--test` and `--help` with the one of `--env`;
* add a comma at the end of the description of `--test`.

**Comment by msimonin** [comment]
What about this ? 
`-` and `--` and text are tab aligned
```
@enostask("""
usage: enos tc [-e ENV|--env=ENV] [-vv|-s|--silent] [--test]

Enforce network constraints

Options:
  -e ENV    --env=ENV   Path to the environment directory. You should
                        use this option when you want to link a specific
                        experiment [default: %s].
            --test      Test the rules by generating various reports.
  -h        --help      Show this help message.
""" % SYMLINK_NAME)
```

**Comment by denaitre** [comment]
Your proposition makes it clearly readable :+1:
However, it would request to adapt the other 8 help messages for consistency (maybe the topic of another PR/issue?).

**Comment by msimonin** [comment]
Yes this should be part of another PR.

**Comment by msimonin** [comment]
pushed

**Pull request merged**